src/status_im2/contexts/quo_preview/avatars/group_avatar.cljs

```diff
@@ -39,7 +42,12 @@
          {:padding-vertical 60
           :flex-direction   :row
           :justify-content  :center}
-         [quo2/group-avatar @state]]]])))
+         (let [{:keys [avatar?]} @state
+               params            (cond-> @state
+                                   (boolean avatar?)
```

Comment:
In Clojure we usually only use `boolean` if we strictly need something to be of the boolean type. In most conditional checks like in cond clauses this is unnecessary, so we can remove it here.